builders/interoperability/xcm/xc-registration/assets.md

```diff
@@ -0,0 +1,431 @@
+    |:-------------------:|:-------------------:|
+    |        Name         |       Glimmer       |
+    |       Symbol        |        GLMR         |
+    |      Decimals       |         18          |
+    | Existential deposit | 1 (1 * 10^-18 GLMR) |
+
+=== "Moonriver"
+    |      Variable       |        Value        |
+    |:-------------------:|:-------------------:|
+    |        Name         |      Moonriver      |
+    |       Symbol        |        MOVR         |
+    |      Decimals       |         18          |
+    | Existential deposit | 1 (1 * 10^-18 MOVR) |
+
+=== "Moonbase Alpha"
+    |      Variable       |       Value        |
+    |:-------------------:|:------------------:|
+    |        Name         |        DEV         |
+    |       Symbol        |        DEV         |
+    |      Decimals       |         18         |
+    | Existential deposit | 1 (1 * 10^-18 DEV) |
+
+The multilocation of Moonbeam native assets includes the parachain ID of the Moonbeam network and the pallet instance where Moonbeam assets live, which corresponds to the index of the Balances Pallet. The multilocation for each network is as follows:
+
+=== "Moonbeam"
+
+    ```js
+    {
+      V3: {
+        parents: 1,
+        interior: {
+          X2: [
+            { 
+              Parachain: 2004
+            },
+            {
+              PalletInstance: 10
+            }
+          ]
+        }
+      }
+    }
+    ```
+
+=== "Moonriver"
+
+    ```js
+    {
+      V3: {
+        parents: 1,
+        interior: {
+          X2: [
+            { 
+              Parachain: 2023
+            },
+            {
+              PalletInstance: 10
+            }
+          ]
+        }
+      }
+    }
+    ```
+
+=== "Moonbase Alpha"
+
+    ```js
+    {
+      V3: {
+        parents: 1,
+        interior: {
+          X2: [
+            { 
+              Parachain: 1000
+            },
+            {
+              PalletInstance: 3
+            }
+          ]
+        }
+      }
+    }
+    ```
+
+### Register Local XC-20s on Another Chain {: #register-local-xc20 }
+
+The multilocation for local XC-20s include the parachain ID of Moonbeam, the pallet instance, and the address of the ERC-20. The pallet instance corresponds to the index of the ERC-20 XCM Bridge Pallet, as this is the pallet that enables any ERC-20 to be transferred via XCM.
+
+**To be registered on other chains, local XC-20s must strictly comply with the standard ERC-20 interface as described in [EIP-20](https://eips.ethereum.org/EIPS/eip-20){target=_blank}. In particular, the [`transfer` function](https://eips.ethereum.org/EIPS/eip-20#transfer){target=_blank} must be as described in EIP-20:**
+
+```js
+function transfer(address _to, uint256 _value) public returns (bool success)
+```
+
+If the function selector of the `transfer` function deviates from the standard, the cross-chain transfer will fail.
+
+You can use the following multilocation to register a local XC-20:
+
```

Comment:
Moonbeam will need to be added once RT240x gets rolled out. There is a separate PR for that, so if this gets merged first I will need to update https://github.com/moonbeam-foundation/moonbeam-docs/pull/687